source/js/components/join/join.jsx

```diff
@@ -311,6 +317,14 @@ export default class JoinUs extends React.Component {
    * Render localization fields
    */
 
+  setCountry(country) {
+    this.setState({ country: country });
```

Comment:
not technically wrong or right, but `{ country: country }` and `{ country }` are equivalent in modern JS.